pp-mock/handlers/transport/v1/transport_search.go

```diff
@@ -74,9 +74,10 @@ func (*TransportSearchV1Server) TransportSearch(ctx context.Context, req *transp
 				}, nil
 			}
 
-			if queryTrip.Departure == nil ||
+			if queryTrip.Departure == nil || queryTrip.Arrival == nil ||
```

Comment:
Now the arrival location is required but the error message if not provided still is 
> Invalid trip: departure must be provided